shard.lock

```diff
@@ -4,29 +4,33 @@ shards:
     git: https://github.com/athena-framework/negotiation.git
     version: 0.1.1
 
+  backtracer:
+    git: https://github.com/sija/backtracer.cr.git
+    version: 1.2.1
+
```

Comment:
What's this new one?

Reply:
> What's this new one?

https://github.com/crystal-loot/exception_page/commit/8f116768f3e2e5ad09dc212176c166d1de1468e9#diff-cbaa8062ee0fcfb5af8adfd98f69af4766b9ad5bb5e79158abe8da34f6fd7de0